content/lua/concepts/mathematical-library/terms/max/max.md

```diff
@@ -0,0 +1,78 @@
+---
+Title: 'max()'
+Description: 'Used to return the maximum value among a set of numerical input values.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+Lua's **`math.max()`** function is a standard function that is a part of the `math` library, and it is also the number-one solution for determining the maximum value from a list of numerical inputs, regardless of the number of values in this list. Whether you're working on a simple arithmetic operation or even a complex mathematical algorithm or data analysis, **`math.max()`** will be a versatile and fundamental helper function for you.
```

Comment:
```suggestion
Lua's **`math.max()`** function is a standard function that is a part of the `math` library, and it is the best solution for determining the maximum value from a list of numerical inputs, regardless of the number of values in this list. Whether the task is just simple arithmetic or a complex mathematical algorithm, `math.max()` is a versatile and fundamental helper function for a variety of cases.
```

Reply:
docs content refrains from using first and second person pronouns (I you we). Additionally, boldface is only used on the first instance of the function